src/Components/LikeButton.jsx

```diff
@@ -1,28 +1,43 @@
-import { useEffect, useState } from "react";
+import { useState } from "react";
 import { updateArticleLikes } from "../api";
-import { useParams } from "react-router-dom";
 
-function LikeButton({ setNewVote }) {
+function Likebutton({ articleId, currentVotes, setNewVote }) {
   const [isLiked, setIsLiked] = useState(false);
-
-  const { article_id } = useParams();
+  const [isError, setIsError] = useState(false);
 
   const handleLike = () => {
+    setIsError(false);
     if (isLiked) {
-      updateArticleLikes(article_id);
-      setNewVote((currentNewVote) => {
-        return currentNewVote - 1;
-      });
+      setNewVote((currentNewVote) => currentNewVote - 1);
       setIsLiked(false);
+      updateArticleLikes(articleId, -1)
+        .then(() => {
+          console.log(`Like removed (${currentVotes})`);
```

Comment:
This will only show in the console. If you'd like to let the users know, alert might be a better choice to use